### DISCUSSION TITLE
[Question]: Should we merge?

### DISCUSSION THREAD

**Discussion by okaufmann** [spatie/laravel-notification-log#6] [Ideas]
Hi, 

There is no Discussion tab, so I created an Issue.

A few months ago I started to creating a log package for Laravel Notifications https://github.com/teamnovu/laravel-notification-log. Today I noticed that you also published one.

Our goal was to be able to track all sent notifications. Especially the message contents. Also, I need to know for how many times a notification was tried to be sent (Some mail servers are really unstable, and we need to retry sending a message several times with some backoff). 

What I like about your package is the simpler implementations and especially the https://github.com/spatie/laravel-notification-log/blob/main/src/Models/Concerns/HasHistory.php trait for checking if a notification was already sent.

If you are interested, I would try to bring in some of our aspects into your package? If not, I would like to copy e.g. the Trait into ours.

Please let me know what you think ✌️

**Comment by freekmurze** [comment]
I'm open for simple PRs that add small features. 

Instead of adding a concrete feature, I prefer PRs that make the package more extensible 👍
